src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -554,7 +553,7 @@ static UCS_CLASS_INIT_FUNC(uct_ud_verbs_iface_t, uct_md_h md, uct_worker_h worke
     self->tx.wr_inl.imm_data          = 0;
     self->tx.wr_inl.next              = 0;
     self->tx.wr_inl.sg_list           = self->tx.sge;
-    self->tx.wr_inl.num_sge           = UCT_UD_MAX_SGE;
+    self->tx.wr_inl.num_sge           = 2;
```

Comment:
@yosefe You requested to remove `UCT_UD_MAX_SGE`. This macro used in one place only. This `wr_inl.num_sge` will be always 2 because it used in `put_short` and `am_short` where header always send as `sge[0]` and payload as `sge[1]`. These numbers (`[1]`) hardcoded.